tests/ui_tools/test_popups.py

```diff
@@ -984,6 +1005,10 @@ def mock_external_classes(
             self.controller, "maximum_popup_dimensions", return_value=(64, 64)
         )
         mocker.patch(LISTWALKER, return_value=[])
+
+        # To show options based on ZFL
+        self.controller.model.server_feature_level = int()
```

Comment:
Can you help me understand why it's odd? Since we are setting the value of the variable where we are testing it, I don't see why the value where it's initialised is relevant. What would you rather set the value as ?

Reply:
Right, you *are* setting the value where it is specifically being used in tests, so that it finds it :+1:

If not specifically being used, does zero make sense? That is, do other tests need the variable, and if so, is zero a good value, or 135, or 137, etc? If it is necessary to explain that, a comment could help.

Regarding being explicit, I meant that `int()` is less clear than simply `0`. Also that a small comment could explain the need for a specific value.